[2020-07-30 22:03:26,092] {{taskinstance.py:616}} INFO - Dependencies all met for <TaskInstance: store_DAG.create_mysql_table 2020-07-29T00:00:00+00:00 [queued]>
[2020-07-30 22:03:26,103] {{taskinstance.py:616}} INFO - Dependencies all met for <TaskInstance: store_DAG.create_mysql_table 2020-07-29T00:00:00+00:00 [queued]>
[2020-07-30 22:03:26,104] {{taskinstance.py:834}} INFO - 
--------------------------------------------------------------------------------
[2020-07-30 22:03:26,105] {{taskinstance.py:835}} INFO - Starting attempt 2 of 2
[2020-07-30 22:03:26,105] {{taskinstance.py:836}} INFO - 
--------------------------------------------------------------------------------
[2020-07-30 22:03:26,124] {{taskinstance.py:855}} INFO - Executing <Task(MySqlOperator): create_mysql_table> on 2020-07-29T00:00:00+00:00
[2020-07-30 22:03:26,125] {{base_task_runner.py:133}} INFO - Running: ['airflow', 'run', 'store_DAG', 'create_mysql_table', '2020-07-29T00:00:00+00:00', '--job_id', '50', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/store_DAG.py', '--cfg_path', '/tmp/tmppiak3hur']
[2020-07-30 22:03:27,282] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table [2020-07-30 22:03:27,281] {{settings.py:213}} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=8354
[2020-07-30 22:03:27,301] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table /usr/local/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2020-07-30 22:03:27,302] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   """)
[2020-07-30 22:03:27,496] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table [2020-07-30 22:03:27,495] {{__init__.py:51}} INFO - Using executor LocalExecutor
[2020-07-30 22:03:27,849] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table [2020-07-30 22:03:27,848] {{dagbag.py:90}} INFO - Filling up the DagBag from /usr/local/airflow/dags/store_DAG.py
[2020-07-30 22:03:27,914] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table [2020-07-30 22:03:27,913] {{cli.py:516}} INFO - Running <TaskInstance: store_DAG.create_mysql_table 2020-07-29T00:00:00+00:00 [running]> on host 6097e8989e8b
[2020-07-30 22:03:27,944] {{mysql_operator.py:61}} INFO - Executing: CREATE TABLE IF NOT EXISTS clean_store_transactions(STORE_ID varchar(50), STORE_LOCATION varchar(50), PRODUCT_CATEGORY varchar(50), PRODUCT_ID int, MRP float, CP float, DISCOUNT float, SP float, DATE date);
[2020-07-30 22:03:27,965] {{taskinstance.py:1047}} ERROR - The conn_id `mysql_conn` isn't defined
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 922, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/mysql_operator.py", line 67, in execute
    parameters=self.parameters)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
    with closing(self.get_conn()) as conn:
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/mysql_hook.py", line 72, in get_conn
    conn = self.get_connection(self.mysql_conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
    conn = random.choice(list(cls.get_connections(conn_id)))
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
    conns = cls._get_connections_from_db(conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
    "The conn_id `{0}` isn't defined".format(conn_id))
airflow.exceptions.AirflowException: The conn_id `mysql_conn` isn't defined
[2020-07-30 22:03:27,968] {{taskinstance.py:1076}} INFO - All retries failed; marking task as FAILED
[2020-07-30 22:03:27,987] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table Traceback (most recent call last):
[2020-07-30 22:03:27,987] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/bin/airflow", line 32, in <module>
[2020-07-30 22:03:27,988] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     args.func(args)
[2020-07-30 22:03:27,988] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2020-07-30 22:03:27,988] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     return f(*args, **kwargs)
[2020-07-30 22:03:27,989] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
[2020-07-30 22:03:27,989] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     _run(args, dag, ti)
[2020-07-30 22:03:27,989] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
[2020-07-30 22:03:27,989] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     pool=args.pool,
[2020-07-30 22:03:27,990] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2020-07-30 22:03:27,990] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     return func(*args, **kwargs)
[2020-07-30 22:03:27,990] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 922, in _run_raw_task
[2020-07-30 22:03:27,990] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     result = task_copy.execute(context=context)
[2020-07-30 22:03:27,990] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/operators/mysql_operator.py", line 67, in execute
[2020-07-30 22:03:27,991] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     parameters=self.parameters)
[2020-07-30 22:03:27,991] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
[2020-07-30 22:03:27,991] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     with closing(self.get_conn()) as conn:
[2020-07-30 22:03:27,991] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/mysql_hook.py", line 72, in get_conn
[2020-07-30 22:03:27,992] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     conn = self.get_connection(self.mysql_conn_id)
[2020-07-30 22:03:27,992] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
[2020-07-30 22:03:27,992] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     conn = random.choice(list(cls.get_connections(conn_id)))
[2020-07-30 22:03:27,992] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
[2020-07-30 22:03:27,992] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     conns = cls._get_connections_from_db(conn_id)
[2020-07-30 22:03:27,993] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2020-07-30 22:03:27,993] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     return func(*args, **kwargs)
[2020-07-30 22:03:27,993] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
[2020-07-30 22:03:27,993] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table     "The conn_id `{0}` isn't defined".format(conn_id))
[2020-07-30 22:03:27,994] {{base_task_runner.py:115}} INFO - Job 50: Subtask create_mysql_table airflow.exceptions.AirflowException: The conn_id `mysql_conn` isn't defined
[2020-07-30 22:03:31,082] {{logging_mixin.py:95}} INFO - [[34m2020-07-30 22:03:31,081[0m] {{[34mlocal_task_job.py:[0m105}} INFO[0m - Task exited with return code 1[0m
message ----> Dependencies all met for <TaskInstance: store_DAG.create_mysql_table 2020-07-29T00:00:00+00:00 [queued]>
message ----> Dependencies all met for <TaskInstance: store_DAG.create_mysql_table 2020-07-29T00:00:00+00:00 [queued]>
message ----> 
--------------------------------------------------------------------------------
message ----> Starting attempt 2 of 2
message ----> 
--------------------------------------------------------------------------------
message ----> Executing <Task(MySqlOperator): create_mysql_table> on 2020-07-29T00:00:00+00:00
message ----> Running: ['airflow', 'run', 'store_DAG', 'create_mysql_table', '2020-07-29T00:00:00+00:00', '--job_id', '56', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/store_DAG.py', '--cfg_path', '/tmp/tmpq9pr65xa']
message ----> Job 56: Subtask create_mysql_table [2020-08-04 21:13:46,408] {{settings.py:213}} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=2879
message ----> Job 56: Subtask create_mysql_table /usr/local/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
message ----> Job 56: Subtask create_mysql_table   """)
message ----> Job 56: Subtask create_mysql_table [2020-08-04 21:13:47,152] {{__init__.py:51}} INFO - Using executor LocalExecutor
message ----> Job 56: Subtask create_mysql_table [2020-08-04 21:13:50,041] {{dagbag.py:90}} INFO - Filling up the DagBag from /usr/local/airflow/dags/store_DAG.py
message ----> Job 56: Subtask create_mysql_table [2020-08-04 21:13:50,246] {{cli.py:516}} INFO - Running <TaskInstance: store_DAG.create_mysql_table 2020-07-29T00:00:00+00:00 [running]> on host 0885e5a330eb
message ----> Executing: CREATE TABLE IF NOT EXISTS clean_store_transactions(STORE_ID varchar(50), STORE_LOCATION varchar(50), PRODUCT_CATEGORY varchar(50), PRODUCT_ID int, MRP float, CP float, DISCOUNT float, SP float, DATE date);
message ----> The conn_id `mysql_conn` isn't defined
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 922, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/mysql_operator.py", line 67, in execute
    parameters=self.parameters)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
    with closing(self.get_conn()) as conn:
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/mysql_hook.py", line 72, in get_conn
    conn = self.get_connection(self.mysql_conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
    conn = random.choice(list(cls.get_connections(conn_id)))
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
    conns = cls._get_connections_from_db(conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
    "The conn_id `{0}` isn't defined".format(conn_id))
airflow.exceptions.AirflowException: The conn_id `mysql_conn` isn't defined
message ----> All retries failed; marking task as FAILED
message ----> Job 56: Subtask create_mysql_table Traceback (most recent call last):
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/bin/airflow", line 32, in <module>
message ----> Job 56: Subtask create_mysql_table     args.func(args)
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
message ----> Job 56: Subtask create_mysql_table     return f(*args, **kwargs)
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
message ----> Job 56: Subtask create_mysql_table     _run(args, dag, ti)
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
message ----> Job 56: Subtask create_mysql_table     pool=args.pool,
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
message ----> Job 56: Subtask create_mysql_table     return func(*args, **kwargs)
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 922, in _run_raw_task
message ----> Job 56: Subtask create_mysql_table     result = task_copy.execute(context=context)
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/operators/mysql_operator.py", line 67, in execute
message ----> Job 56: Subtask create_mysql_table     parameters=self.parameters)
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
message ----> Job 56: Subtask create_mysql_table     with closing(self.get_conn()) as conn:
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/mysql_hook.py", line 72, in get_conn
message ----> Job 56: Subtask create_mysql_table     conn = self.get_connection(self.mysql_conn_id)
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
message ----> Job 56: Subtask create_mysql_table     conn = random.choice(list(cls.get_connections(conn_id)))
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
message ----> Job 56: Subtask create_mysql_table     conns = cls._get_connections_from_db(conn_id)
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
message ----> Job 56: Subtask create_mysql_table     return func(*args, **kwargs)
message ----> Job 56: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
message ----> Job 56: Subtask create_mysql_table     "The conn_id `{0}` isn't defined".format(conn_id))
message ----> Job 56: Subtask create_mysql_table airflow.exceptions.AirflowException: The conn_id `mysql_conn` isn't defined
message ----> [[34m2020-08-04 21:13:54,418[0m] {{[34mlocal_task_job.py:[0m105}} INFO[0m - Task exited with return code 1[0m
